Test/task2.c

```diff
@@ -0,0 +1,117 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#define testLength 5
+
+void selectionSortEven(int array[], int length);
+bool isSortedInEven(int array[], int length);
+bool tests();
+
+int main()
+{
+    if(!tests())
+    {
+        return 1;
+    }
+    
+    int size = 0;
+
+    printf("Enter the size of the array: ");
+    scanf("%d", &size);
+
+    while(size <= 0)
+    {
+        printf("Invalid input (enter a natural number)");
+        printf("Enter the size of the array: ");
+        scanf("%d", &size);
+    }
+    
+    int *array = malloc(sizeof(int) * size);
+    if(array == NULL)
+    {
+        printf("Memory is not avaialble!\n");
+        return 1;
+    }
+
+    for(int i = 0; i < size; i++)
+    {
+        printf("Enter the element #%d: ", i);
+        scanf("%d", &array[i]);
+    }
+    
+    selectionSortEven(array, size);
+    
+    printf("Sorted array: ");
+    
+    for(int i = 0; i < size; i++)
+    {
+        printf("%d ",array[i]);
+    }
+    
+    free(array);
+    return 0;
+}
+
+void selectionSortEven(int array[], int length)
+{
+    for(int i = 0; i < length - 1; i += 2)
+    {
+        int minIndex = i;
+        for(int j = i + 2; j < length; j+= 2)
+        {
+            if(array[j] < array[minIndex])
+            {
+                minIndex = j;
+            }
+            if(i != minIndex)
+            {
+                int temp = array[i];
+                array[i] = array[minIndex];
+                array[minIndex] = temp;
+            }
+        }
+    }
+}
+
+bool isSortedInEven(int array[], int length)
+{
+    
+    for(int i = 0; i < length - 2; i += 2)
```

Comment:
```suggestion
{
    for(int i = 0; i < length - 2; i += 2)
```
После `{` пустая строка не ставится